app/assets/javascripts/ama_validators/country_province_selector.js.coffee.erb

```diff
@@ -0,0 +1,33 @@
+class window.CountryProvinceSelector
```

Comment:
Could you put classes/functions under an AMA namespace?
https://github.com/amaabca/membership/blob/master/app/assets/javascripts/amanamespace.js.coffee